scripts/release/utils.js

```diff
@@ -19,27 +20,36 @@ function fitTerminal(input) {
   return input;
 }
 
-function logPromise(name, promise) {
+async function logPromise(name, promise) {
+  if (typeof promise === "function") {
+    promise = promise();
+  }
```

Comment:
This variable name is hard to understand to me. So what do you think about the below suggestion?

```suggestion
async function logPromise(name, promiseOrAsyncFunction) {
  const promise =
    typeof promiseOrAsyncFunction === "function"
      ? promise()
      : promiseOrAsyncFunction;
```